### DISCUSSION TITLE
Following Argo Events Tutorial documentation

### DISCUSSION THREAD

**Discussion by nheitz** [argoproj/argo-events#1253] [Q&A]
I'm sure the answer will seem really straightforward, but I can't seem to find relevant documentation.

The tutorial here:
https://argoproj.github.io/argo-events/tutorials/01-introduction/

states this:
> Make sure to configure Argo Workflow controller to listen to workflow objects created in argo-events namespace. 

But I can't find the relevant documentation for how to do this!  I have followed basic setup for both argo and argo events based on latest docs, but the webhooks example does not appear to be triggering workflows.  I think the above allusion to "listen for workflow objects" must be the culprit.  Any help?

**Comment by tpiperatgod** [comment]
Hi @nheitz, I had the same question: the webhook instance that was triggered was not found after executing the [Webhook](https://argoproj.github.io/argo-events/quick_start/#getting-started) example.

Later I checked and found that although the webhook Sensor was created, its replications of deployments was not ready. After checking the logs, I found that the reason was that I had not created a Service Account named **`operate-workflow-sa`**.

Once I created the relevant sa, I was able to run the Webhook example.

I suggest you to check whether the relevant components (Especially the Sensor) is ready or not, and if there is an exception in the component, you can determine the approximate cause of the problem by its logs.
